meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -133,6 +133,7 @@ RESIN_CONFIGS[docker] ?= " \
     CONFIG_EXT4_FS_SECURITY=y \
     CONFIG_AUFS_FS=y \
     CONFIG_AUFS_XATTR=y \
+    CONFIG_KEYS=y \
```

Comment:
It's strange because I had checked on a rpi3 and this config is not there:

# cat /proc/config.gz | grep -i CONFIG_KEYS
# 
(so no output from the above command, nothing found)